Resources/Prototypes/Stray/Flavors/flavors.yml

```
@@ -1,4 +1,9 @@
 - type: flavor
   id: sawdustsoup
   flavorType: Complex
-  description: flavor-complex-frostpunk
\ No newline at end of file
+  description: flavor-complex-frostpunk
+  
+- type: flavor
+  id: flavor-tarragon
+  flavorType: Complex
+  description: flavor-tarragon
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)